avocado/utils/process.py

```diff
@@ -1303,3 +1300,24 @@ def get_command_output_matching(command, pattern):
     :rtype: list of str
     """
     return [line for line in run(command).stdout_text.splitlines() if pattern in line]
+
+
+def get_ping_process_pid(interface="", peer_ip="", ping_count=""):
+    """
+    Funtion to get the process ID of ping flood.
+
+    :returns : Process PID number that initated by ping flood.
+    :rtype : int
+    """
+    cmd = (
+        f"ps -ef | grep '[p]ing -I {interface} {peer_ip} -c {ping_count} -f' "
+        f"| awk '{{print $2}}' | head -1"
```

Comment:
can this not be handle via user (Test writer) i did not feel we should be having utility for this

Reply:
@PraveenPenguin yes this should be handled in test itself, as this is not generic ping @FarooqAbdulla01 please close this PR

Reply:
Closing this PR, as the code support added from test script itself